README.md

```diff
@@ -37,39 +35,101 @@ group :development, :test do
 end
 ```
 
-Generators for factories will automatically substitute fixture (and maybe any other
-`fixture_replacement` you set). If you want to disable this feature, add the
-following to your application.rb file:
+You may want to configure your test suite to include factory\_bot methods; see
+[configuration](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#configure-your-test-suite).
```

Comment:
Is there is a way to make this a relative link, so e.g. if someone is on a different branch/tag, they don't get routed to the master documentation?

Reply:
It needs to work on RubyDocs as well, and I haven't found a good way to make relative links work there.